modules/ztp-creating-siteconfig-custom-resources.adoc

```diff
@@ -0,0 +1,351 @@
+--from=quay.io/openshift-release-dev/ocp-release:{{ mirror_version_spoke_release }}
+--to={{ provisioner_cluster_registry }}/ocp4 --to-release-image={{
+provisioner_cluster_registry }}/ocp4:{{ mirror_version_spoke_release }}
+----
+
+* You need to mirror the ISO and `rootfs` used to generate the spoke cluster ISO. Store the images in an http server and configure the settings to pull images from there.
++
+The images must match the version of the ClusterImageSet. So, to deploy a 4.8.0 version, the `rootfs` and ISO need to be 4.8.0 as well.
+
+
+.Procedure
+
+Use this procedure to create and provision one single node cluster. To provision additional clusters, repeat this procedure for each cluster.
+
+. Create the namespace for the single node managed cluster:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+     name: <cluster-name> <1>
+     labels:
+        name: <cluster-name> <1>
+----
+<1>  `cluster-name` is the name of the managed single node cluster to provision.
+
+. Create the BMC Secret custom resource:
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  password: <bmc-password> <1>
+  username: <bmc-username> <2>
+kind: Secret
+metadata:
+  name: <cluster-name>-bmc-secret
+  namespace: <cluster-name>
+type: Opaque
+----
+<1> `bmc-password` is the password to the target bare metal machine. Must be base-64 encoded.
+<2> `bmc-username` is the username to the target bare metal machine. Must be base-64 encoded.
+
+. Create the Image Pull Secret custom resource:
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  .dockerconfigjson: <pull-secret> <1>
+kind: Secret
+metadata:
+  name: assisted-deployment-pull-secret
+  namespace: <cluster-name>
+type: kubernetes.io/dockerconfigjson
+----
+<1> `pull-secret` is the Red Hat OpenShift Container Platform pull secret. Must be base-64 encoded.
+
+. Create the AgentClusterInstall custom resource:
++
+[source,yaml]
+----
+apiVersion: extensions.hive.openshift.io/v1beta1
+kind: AgentClusterInstall
+metadata:
+  # Only include the annotation if using OVN, otherwise omit the annotation
+  annotations:
+    agent-install.openshift.io/install-config-overrides: '{"networking":{"networkType":"OVNKubernetes"}}'
+  name: <cluster-name>
+  namespace: <cluster-name>
+spec:
+  clusterDeploymentRef:
+    name: <cluster-name>
+  imageSetRef:
+    name: <cluster-image-set> <1>
+  networking:
+    clusterNetwork:
+    - cidr: 10.128.0.0/14
+      hostPrefix: 23
+    machineNetwork:
+    - cidr: <machine-network-cidr> <2>
+    serviceNetwork:
+    - 172.30.0.0/16
+  provisionRequirements:
+    controlPlaneAgents: 1
+    workerAgents: 0
+  sshPublicKey: <public-key> <3>
+----
++
+<1> `cluster-image-set` is the name of the ClusterImageSet resource chosen in the prerequisite step.
+<2> `machine-network-cidr` is the target bare metal machine’s CIDR.
+<3> `public-key` entered as plain text can be used to SSH into the target bare metal machine after the Host is installed.
```

Comment:
I think Host does not need to be capitalized here. If I am going with guidance from https://redhat-documentation.github.io/supplementary-style-guide/#_how_to_use_this_guide ""Host name" is the preferred spelling; do not use "hostname." Only capitalize the initial "H" at the beginning of a sentence." I assume the same applies to Host